twine/commands/register.py

```diff
@@ -23,8 +23,21 @@
 from twine import utils
 
 
-def register(package, repository, username, password, comment, config_file,
-             cert, client_cert, repository_url):
+def register(
+        package,
+        repository="pypi",
+        username=None,
+        password=None,
+        comment=None,
+        config_file="~/.pypirc",
+        cert=None,
+        client_cert=None,
+        repository_url=None,
+):
+    # IMPORTANT: Before version 2.0 there were only arguments here, not keywork
```

Comment:
I should have explained. This was a preliminary guess that this might end up in a future version 2.0.